packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DateRangeFilter.test.tsx

```diff
@@ -9,6 +9,8 @@ import {
   DateRangeFilter,
   type DateRangeFilterProps,
 } from './DateRangeFilter.js';
+import userEvent from '@testing-library/user-event';
+import { assertMonthYear } from '../../../DatePicker/DatePicker.test.js';
```

Comment:
we generally don't import files from one test into another.

i think in this test you don't need to assert the whole thing (DatePicker's implementation details don't need to be tested). just a simple sanity test should be enough; for example:
```js
getByText('January-custom');
getByText('Su-custom');
getByText('Sunday-custom');
```

Reply:
Thanks for let me know! I cleaned up the test and removed the export